Source/Core/VideoBackends/OGL/VertexManager.cpp

```diff
@@ -156,8 +158,20 @@ void VertexManager::vFlush()
   // setup the pointers
   nativeVertexFmt->SetupVertexPointers();
 
+  if (!g_Config.backend_info.bSupportsFragmentStoresAndAtomics && ::BoundingBox::active)
+  {
+    glEnable(GL_STENCIL_TEST);
+    OGL::BoundingBox::stencil_updated = true;
```

Comment:
Given this is the only place these are used externally, it would probably be preferable to make this value setting part of `BoundingBox`'s API via a function, rather than exposing public variables. (e.g. BoundingBox::UpdateStencilState or something more appropriate if you know of one).